tokens/src/components/Form/typography.json

```diff
@@ -0,0 +1,66 @@
+{
+  "typography": {
+    "form": {
+      "input": {
+        "font": {
+          "family": { "value": "{input.btn.font.family.value}", "source": "$input-font-family" },
+          "size": {
+            "base": { "value": "{input.btn.font.size.base.value}", "source": "$input-font-size" },
+            "sm": { "value": "{input.btn.font.size.sm.value}", "source": "$input-font-size-sm" },
+            "lg": { "value": "{input.btn.font.size.lg.value}", "source": "$input-font-size-lg" }
+          },
+          "weight": { "value": "{font.weight.base.value}", "source": "$input-font-weight" }
+        },
+        "line-height": {
+          "base": { "value": "{input.btn.line.height.base.value}", "source": "$input-line-height" },
+          "sm": { "value": "{input.btn.line.height.sm.value}", "source": "$input-line-height-sm" },
+          "lg": { "value": "{input.btn.line.height.lg.value}", "source": "$input-line-height-lg" }
+        }
+      },
+      "control": {
+        "label": {
+          "floating": {
+            "text": { 
+              "value": "{color.form.input.bg.base.value}",
```

Comment:
this should probably be in `color` category

Reply:
done, thx